src/components/Navbar/NavbarBody/ProfileAvatarButton.tsx

```
@@ -127,10 +147,13 @@ const ProfileAvatarButton = () => {
     <Button
       tooltip={t('sign-in')}
       ariaLabel={t('sign-in')}
-      variant={ButtonVariant.Simplified}
+      variant={ButtonVariant.SimplifiedAccent}
       size={ButtonSize.Small}
       href={getLoginNavigationUrl(router.asPath)}
-      onClick={onTriggerClicked}
+      onClick={() => {
+        dispatch({ type: setIsNavigationDrawerOpen.type, payload: false });
+        onTriggerClicked;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(no-unused-expressions)